data/src/main/kotlin/com/mashup/dorabangs/data/di/RepositoryModule.kt

```diff
@@ -32,4 +34,10 @@ abstract class RepositoryModule {
     abstract fun bindDoraUrlCheckRepository(
         repository: DoraUrlCheckRepositoryImpl,
     ): DoraUrlCheckRepository
+
+    @Binds
+    @Reusable
+    abstract fun bindPostsRepository(
```

Comment:
위에 나도 그렇게 쓰긴했는데 . ㅋㅋ. 
provides로 썼음 binds가 맞지 않을까요 ?! 나도 똑같이 썻으니 내꺼도 바꺼조
```suggestion
    abstract fun bindsPostsRepository(
```